pkg/controllers/management/restrictedadminrbac/register.go

```diff
@@ -51,7 +51,6 @@ func Register(ctx context.Context, management *config.ManagementContext, wrangle
 		provClusters:        wrangler.Provisioning.Cluster().Cache(),
 	}
 
-	r.clusters.AddHandler(ctx, "restrictedAdminsRBACCluster", r.clusterRBACSync)
 	r.projects.AddHandler(ctx, "restrictedAdminsRBACProject", r.projectRBACSync)
```

Comment:
Do we need to explicitly give the restricted-admin permissions if they are a cluster-owner?

Reply:
I don't think so, since we already explicitly create a cluster role binding to the `cluster-admin` cluster role downstream in two situations: when we get a new cluster and when we get a new admin.

Reply:
Sorry, The comment was for line `r.projects.AddHandler(ctx, "restrictedAdminsRBACProject", r.projectRBACSync)`
This should no longer be needed if the restricted-admin is a cluster-owner.

Reply:
Did some digging, and this handler does the following on project change
1. If the project belongs to the local cluster: return
2. Get all of the GRBs to restricted admin
3.  For each GRB to restricted admin:
4.  ensure there is a roleBinding for the GRB in the projects  namespace to "project-crd-clusterRole" AKA `*,  *` in `management.cattle.io` & `provisioning.cattle.io`

While cluster-owner grants `*` for the following resource in the local cluster for the project namespace
``` golang
var projectManagmentPlaneResources = map[string]string{
	"apps":                        "project.cattle.io",
	"apprevisions":                "project.cattle.io",
	"catalogtemplates":            "management.cattle.io",
	"catalogtemplateversions":     "management.cattle.io",
	"pipelines":                   "project.cattle.io",
	"pipelineexecutions":          "project.cattle.io",
	"pipelinesettings":            "project.cattle.io",
	"sourcecodeproviderconfigs":   "project.cattle.io",
	"projectloggings":             "management.cattle.io",
	"projectalertrules":           "management.cattle.io",
	"projectalertgroups":          "management.cattle.io",
	"projectcatalogs":             "management.cattle.io",
	"projectmonitorgraphs":        "management.cattle.io",
	"projectroletemplatebindings": "management.cattle.io",
	"secrets":                     "",
}
```

Reply:
@MbolotSuse @cmurphy Do either of you know if a restricted admin needs more permissions than the ones granted by the above `projectManagmentPlaneResources` map?